### DISCUSSION TITLE
Upload of a CSV file to create student accounts for a class

### DISCUSSION THREAD

**Discussion by stevecopley** [hedyorg/hedy#3933] [Website]
When I'm using Scratch with a class, to avoid too much faffing about, I usually create the student account ahead of the first lesson. Scratch has the ability to upload a **CSV file** with a list of usernames/passwords to mass-create the accounts and link them to the current class. Something similar for Hedy would be great.

(Would be good to have a downloadable template CSV file with a couple of examples in to give teachers clear guidance as to the fields/formats you required.)

**Comment by Felienne** [comment]
Lovely idea! I made an issue: #3936
